resolwe/observers/views.py

```diff
@@ -0,0 +1,55 @@
+"""Background task viewset."""
+
+from rest_framework import mixins, serializers, viewsets
+
+from resolwe.observers.mixins import ObservableMixin
+from resolwe.observers.models import BackgroundTask
+
+STATUSES = [status[0] for status in BackgroundTask.STATUS_CHOICES]
+
+
+class BackgroundTaskOptions(serializers.Serializer):
+    """Used to deserialize options set no BackgrondTaskViewSet."""
+
+    with_superuser = serializers.BooleanField(default=False, required=False)
```

Comment:
Does it make sense to expose this option? I think we don't do it anywhere else.

Reply:
OK, I have remove it, as it simplifies the code.
As a side effect superusers will see everybody's background tasks.